README.md

```diff
@@ -170,7 +171,7 @@ Command flag              | Description
 `--except`                | Run all cops enabled by configuration except the specified cop(s) and/or departments.
 `--auto-gen-config`       | Generate a configuration file acting as a TODO list.
 `--show-cops`             | Shows available cops and their configuration.
-`--fail-level`            | Minimum severity for exit with error code.
+`--fail-level`            | Minimum [severity](#severity) for exit with error code. Full severity name or upper case initial can be given. Use `A` or `autocorrect` to fail when all offenses were auto-corrected.
```

Comment:
I think this is not clear enough. Something in the lines of "Normally auto-corrected offenses are ignored by flag. Use `A` or `autocorrect` if you'd like them to still trigger failure.". Not sure about the exact wording.